utils/constants.jsx

```diff
@@ -1319,6 +1319,7 @@ export const Constants = {
         swift: {name: 'Swift', extensions: ['swift']},
         tex: {name: 'TeX', extensions: ['tex'], aliases: ['latex']},
         text: {name: 'Text', extensions: ['txt', 'log']},
+        typescript: {name: 'TypeScript', extensions: ['ts'], aliases: ['ts', 'tsx']},
```

Comment:
If we're including `tsx` as an alias, does this also support highlighting TypeScript with React embedded in it? If so, we should also include `tsx` as an extension so that `tsx` files are highlighted when viewed as file attachments.

Reply:
That sounds completely reasonable. :grin: